cmsdb/campaigns/run3_2023_preBPix_nano_v12/qcd.py

```diff
@@ -0,0 +1,410 @@
+        ),
+    ),
+)
+cpn.add_dataset(
+    name="qcd_em_pt30to50_pythia",
+    id=14790908,
+    processes=[procs.qcd_em_pt30to50],
+    info=dict(
+        nominal=DatasetInfo(
+            keys=[
+                "/QCD_PT-30to50_EMEnriched_TuneCP5_13p6TeV_pythia8/Run3Summer23NanoAODv12-130X_mcRun3_2023_realistic_v14-v2/NANOAODSIM",  # noqa
+            ],
+            n_files=10,
+            n_events=2053805,
+        ),
+    ),
+)
+cpn.add_dataset(
+    name="qcd_em_pt50to80_pythia",
+    id=14790503,
+    processes=[procs.qcd_em_pt50to80],
+    info=dict(
+        nominal=DatasetInfo(
+            keys=[
+                "/QCD_PT-50to80_EMEnriched_TuneCP5_13p6TeV_pythia8/Run3Summer23NanoAODv12-130X_mcRun3_2023_realistic_v14-v2/NANOAODSIM",  # noqa
+            ],
+            n_files=9,
+            n_events=1990713,
+        ),
+    ),
+)
+cpn.add_dataset(
+    name="qcd_em_pt80to120_pythia",
+    id=14789970,
+    processes=[procs.qcd_em_pt80to120],
+    info=dict(
+        nominal=DatasetInfo(
+            keys=[
+                "/QCD_PT-80to120_EMEnriched_TuneCP5_13p6TeV_pythia8/Run3Summer23NanoAODv12-130X_mcRun3_2023_realistic_v14-v2/NANOAODSIM",  # noqa
+            ],
+            n_files=9,
+            n_events=2076482,
+        ),
+    ),
+)
+cpn.add_dataset(
+    name="qcd_em_pt120to170_pythia",
+    id=14790533,
+    processes=[procs.qcd_em_pt120to170],
+    info=dict(
+        nominal=DatasetInfo(
+            keys=[
+                "/QCD_PT-120to170_EMEnriched_TuneCP5_13p6TeV_pythia8/Run3Summer23NanoAODv12-130X_mcRun3_2023_realistic_v14-v2/NANOAODSIM",  # noqa
+            ],
+            n_files=10,
+            n_events=2100319,
+        ),
+    ),
+)
+cpn.add_dataset(
+    name="qcd_em_pt170to300_pythia",
+    id=14790618,
+    processes=[procs.qcd_em_pt170to300],
+    info=dict(
+        nominal=DatasetInfo(
+            keys=[
+                "/QCD_PT-170to300_EMEnriched_TuneCP5_13p6TeV_pythia8/Run3Summer23NanoAODv12-130X_mcRun3_2023_realistic_v14-v2/NANOAODSIM",  # noqa
+            ],
+            n_files=9,
+            n_events=2141399,
+        ),
+    ),
+)
+cpn.add_dataset(
+    name="qcd_em_pt300toinf_pythia",
+    id=14790134,
+    processes=[procs.qcd_em_pt300toinf],
+    info=dict(
+        nominal=DatasetInfo(
+            keys=[
+                "/QCD_PT-300_EMEnriched_TuneCP5_13p6TeV_pythia8/Run3Summer23NanoAODv12-130X_mcRun3_2023_realistic_v14-v2/NANOAODSIM",  # noqa
+            ],
+            n_files=9,
+            n_events=1960617,
+        ),
+    ),
+)
+# cpn.add_dataset(
+#     name="qcd_em_pt300toinf_pythia",
+#     id=14790606,
+#     processes=[procs.qcd_em_pt300toinf],
+#     info=dict(
+#         nominal=DatasetInfo(
+#             keys=[
+#                 "/QCD_PT-300toInf_EMEnriched_TuneCP5_13p6TeV_pythia8/Run3Summer23NanoAODv12-130X_mcRun3_2023_realistic_v14-v2/NANOAODSIM",  # noqa
+#             ],
+#             n_files=9,
+#             n_events=1582951,
+#         ),
+#     ),
+# )
```

Comment:
What's the difference to the dataset above? If we're sure that the one above is correct, I'd propose to delete this part here

```suggestion
```

Reply:
Thank you for the review! I'm not sure where the difference is between the two datasets (except for the name), but the one that I kept had more MC stats compared to this one. But I have no preferences on which one should be kept